File: app/src/main/java/com/example/myapplication/javakotlinpractical/FlowControl.java
package com.example.myapplication.javakotlinpractical;

public class FlowControl {
    public static void main(String arg[]) {
        //ifelse ladder
        int time = 22;
        if (time < 10) {
            System.out.println("Good morning.");
        } else if (time < 20) {
            System.out.println("Good day.");
        } else {
            System.out.println("Good evening.");
        }

        int i = 0;
        //while
        while (i < 5) {
            System.out.println(i);
            i++;
        }
        //for loop
        for (int j = 0; j <= 10; j = j + 2) {
            System.out.println(j);
        }
        //int i = 0;
        while (i < 10) {
            if (i == 4) {
                i++;
                continue;
            }
            System.out.println(i);
            i++;
        }
    }
}

File: app/src/main/java/com/example/myapplication/javakotlinpractical/FlowControl.kt
package com.example.myapplication.javakotlinpractical

import java.util.*;

fun main(args: Array<String>) {
    //ifelse
    println("If else")
    val time = 22
    if (time < 10) {
        println("\nGood morning.")
    } else if (time < 20) {
        println("Good day.")
    } else {
        println("Good evening.")
    }

//when similar to switch
    var number = 4
    println("\nWhen")
    var numberProvided = when(number) {
        1 -> "One"
        2 -> "Two"
        3 -> "Three"
        4 -> "Four"
        5 -> "Five"
        else -> "invalid number"
    }
    println("You provide $numberProvided")

    //dowhile
    println("\nDowhile")
    var i = 1
    do {
        println(i)
        i++
    }
    while (i<=5);

    println("for loop")
    val marks = arrayOf(80,85,60,90,70)
    for(item in marks) {
        println(item)
    }
}









